src/Provider/3dTilesProvider.js

```diff
@@ -130,8 +130,21 @@ function b3dmToMesh(data, layer, url) {
     });
 }
 
-function pntsParse(data) {
-    return PntsParser.parse(data).then(result => ({ object3d: result.point }));
+function pntsParse(data, layer) {
+    return PntsParser.parse(data).then((result) => {
+        const material = layer.material ?
```

Comment:
the overload of the material by the layer, is it documenting?

Reply:
No it's not documented.

IMHO this part of iTowns is still moving, and should be properly documented when the refactoring regarding styling is done. WDYT?

Reply:
we must add a minimum of documentation
and why not just pass the material?

Reply:
> we must add a minimum of documentation

Sure. But we currently we don't have any place to add this documentation. I might start working on something like this, but this PR is definitely not the place to start documenting layers.

> we must add a minimum of documentation

What do you mean?